packages/build-utils/src/fs/run-user-scripts.ts

```diff
@@ -234,7 +236,13 @@ export async function getNodeVersion(
 
 export async function scanParentDirs(
   destPath: string,
-  readPackageJson = false
+  {
+    readPackageJson = false,
+    spawnOpts,
+  }: {
+    readPackageJson?: boolean;
+    spawnOpts?: SpawnOptions;
+  } = {}
```

Comment:
We can't introduce breaking changes to build-utils because it gets injected for 3rd party builders.

So this will need to support the old api signature too